install_config/storage_examples/storage_classes_dynamic_provisioning.adoc

```diff
@@ -0,0 +1,322 @@
+pvc-a9f70544-8bfd-11e6-9962-42010af00004   5Gi        RWX           Delete          Bound     rh-eng/pvc-engineering2             5m <1>
+pvc-ba4612ce-8b4d-11e6-9962-42010af00004   5Gi        RWO           Delete          Bound     mytest/gce-dyn-claim1               21h
+pvc-e9b4fef7-8bf7-11e6-9962-42010af00004   10Gi       RWX           Delete          Bound     rh-eng/pvc-engineering              46m <2>
+
+----
+<1> This PV was bound to our _default_ dynamic volume from the _default_ StorageClass.
+<2> This PV was bound to our first PVC from example 1 with our _fast_ StorageClass.
+====
+
+Create a manual disk in the GCE console (not dynamically provisioned) and a Persistent Volume that connects to the new GCE disk.
+
+Now view the PVs again, notice we have a pv-manual-gce volume Available.
+
+====
+----
+
+# oc get pv
+NAME                                       CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS      CLAIM                     REASON    AGE
+pv-manual-gce                              35Gi       RWX           Retain          Available                                       4s
+pvc-a9f70544-8bfd-11e6-9962-42010af00004   5Gi        RWX           Delete          Bound       rh-eng/pvc-engineering2             12m
+pvc-ba4612ce-8b4d-11e6-9962-42010af00004   5Gi        RWO           Delete          Bound       mytest/gce-dyn-claim1               21h
+pvc-e9b4fef7-8bf7-11e6-9962-42010af00004   10Gi       RWX           Delete          Bound       rh-eng/pvc-engineering              53m
+
+----
+====
+
+Now create another claim identical to the PVC above but change the name (no annotation set). Will the claim get satisfied by the _default_ StorageClass or the manually available Volume?
+
+
+.Claim Spec Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+ name: pvc-engineering3
+spec:
+ accessModes:
+  - ReadWriteMany
+ resources:
+   requests:
+     storage: 15Gi
+
+----
+====
+
+
+Even though the claim was satisfied by the existing and available _pv-manual-gce_ PV, because _default_ StorageClass was enabled, a volume was dynamically provisioned based on the claim request.
+
+====
+----
+
+# oc get pvc
+NAME               STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+pvc-engineering    Bound     pvc-e9b4fef7-8bf7-11e6-9962-42010af00004   10Gi       RWX           1h
+pvc-engineering2   Bound     pvc-a9f70544-8bfd-11e6-9962-42010af00004   5Gi        RWX           19m
+pvc-engineering3   Bound     pvc-6fa8e73b-8c00-11e6-9962-42010af00004   15Gi       RWX           6s
+
+----
+==== 
+
+
+[IMPORTANT]
+====
+Once the _default_ StorageClass is enabled, and there are no additional labels/selectors or prebound characteristics associated with a claim and volume, all requests with the exception of an actual existing StorageClass annotation (like in example 1) will be dynamically provisioned
```

Comment:
This is not true. What is true is that if you create a default storage class you need to create your manual PVs in that storage class if you want them to be bound by default.